src/client/Client.cc

```diff
@@ -2402,6 +2408,7 @@ void Client::handle_client_session(const MConstRef<MClientSession>& m)
       if (session->state == MetaSession::STATE_OPEN) {
         ldout(cct, 10) << "mds." << from << " already opened, ignore it"
                        << dendl;
+	reinit_mds_features(session.get(), m);
```

Comment:
Can we add a comment why this is necessary?

Reply:
Sure!

Reply:
done.